osg-run-tests

```diff
@@ -89,33 +109,13 @@ RUN_DIR_NAME=`date '+run-%Y%m%d-%H%M'`
 RUN_DIRECTORY=$TEST_HOME/$RUN_DIR_NAME
 echo "Run directory: $RUN_DIRECTORY"
 
-# Make the new run directory
-safe_run mkdir $RUN_DIRECTORY
-
-# Check out files to a temporary directory
-TEMP_DIR=`mktemp --directory`
-GIT_CHECKOUT_DIR=$TEMP_DIR/git-files
-safe_run git clone --quiet --depth 1 https://github.com/opensciencegrid/vm-test-runs $GIT_CHECKOUT_DIR
-
-# Copy needed files to the run directory
-cd $GIT_CHECKOUT_DIR
-safe_run cp -r \
-    master-run.dag \
-    generate-dag.sub generate-dag osg-test.conf test-exceptions.yaml component-tags.yaml \
-    create-io-images.sub create-io-images \
-    run-job osg-test.patch test-changes.patch osg-release.patch single-test-run.sub true.sub inner-dag.config \
-    process-job-output extract-job-output analyze_job_output.py \
-    combine-job-analyses analyze-test-run.sub analyze-test-run email-analysis vmu.py \
-    report-job-failures report-job-failures.sub \
-    vmu-reporter html-report.sub taglib.py \
-    upload-job-output upload-job-output.sub \
-    $RUN_DIRECTORY/
+# Prepare the run directory
+safe_run git clone --quiet --depth 1 --branch "$GH_BRANCH" git://github.com/"$GH_ACCOUNT"/vm-test-runs.git $RUN_DIRECTORY
 
 if [ $PRERELEASE -eq 1 ]; then
     echo "Using parameters for prerelease tests"
-    safe_run cp -r parameters.d-prerelease $RUN_DIRECTORY/parameters.d
-else
-    safe_run cp -r parameters.d $RUN_DIRECTORY/
+    rm -f $RUN_DIRECTORY/parameters.d/*.yaml
+    safe_run cp -r $RUN_DIRECTORY/parameters.d-prerelease $RUN_DIRECTORY/parameters.d
```

Comment:
Does this prerelease stuff work right for you?  (I have not tested the changes.)

Reply:
No it does not, good catch

Reply:
Ok, yeah i think the previous behavior was copying in the entire parameters.d dir (regular or prerelease), which it had omitted earlier in the big list of stuff to copy; but now since the dir exists already from the git checkout, just the files need to be copied.